appveyorDashboard.yml

```diff
@@ -0,0 +1,15 @@
+install:
```

Comment:
https://ci.appveyor.com/project/mamaso/azure-webjobs-sdk/history

It's on my account so I wasn't tying up build server time